tests/unittests/tests/01-run.py

```diff
@@ -23,7 +24,7 @@ def main():
         # make reset yields error on some boards even if successful
         pass
     try:
-        child.expect(r"OK \([0-9]+ tests\)")
+        child.expect(u"OK \\([0-9]+ tests\\)")
```

Comment:
((Only the sequence `ur"…"` was dropped in Python3, because Python2's `ur"\u1234" == u"\u1234"` (i.e. `\u` is interpreted) was (rightly) deemed as confusing.))